cli/connector/opts.go

```diff
@@ -66,28 +68,27 @@ func MakeConnectOpts(connString, username, password string,
 
 	addrLen := len(address)
 	switch {
-	case addrLen > 0 && (address[0] == '.' || address[0] == '/'):
+	case addrLen > 0 && (address[0] == '.' || address[0] == '/' || address[0] == '~'):
```

Comment:
In general it looks like violating the Single Responsibility Principle: we have 2 places where we process URI.

I suggest to do it at `cli/cmd/connect.go` only and don't try to parse URI at `MakeConnectOpts`.

1. We need to move the parse logic into `cli/cmd/connect.go`.
2. We could remove `MakeConnectOpts` and initialize the struct object like this:
```Go
ConnectOpts{
    Network: UnixNetwork,
    Address: address,
    User: user,
    Password: pass,
    SslOpts: sslOpts,
}
``` 
It is just a suggestion. Up to you.

Reply:
It's nice point, parsing logic and tests were moved to `cli/cmd/connect.go`. 
Also, `resolveConnectOpts` now returns `ConnectOpts` and the remaining args (instead of replacing first argument to the URI).